lib/vector-core/src/tls/mod.rs

```diff
@@ -100,6 +100,8 @@ pub enum TlsError {
     AddCertToStore { source: ErrorStack },
     #[snafu(display("Error setting up the verification certificate: {}", source))]
     SetVerifyCert { source: ErrorStack },
+    #[snafu(display("Error setting SNI: {}", source))]
```

Comment:
Could you add `sni` to `.github/actions/spelling/expect.txt`? I tried but didn't have permissions to push to this branch.

Reply:
added.